client/__tests__/AddReviewModal.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen, cleanup, fireEvent } from '@testing-library/react';
+
+import ProductDetailPage from '../src/components/ProductDetailPage.jsx';
+
+beforeEach(() => {
+  var ProductDetailPageMock = new ProductDetailPage;
```

Comment:
this is slick, I was trying to figure out how to test my modal the other day and this is the way. 👌👌👌

Reply:
This was Sam's solution. I was having trouble accessing the top-level methods using the enzyme mount() method so I reverted back to this.